### ISSUE TITLE
When will this be ready?

### ISSUE THREAD

**Issue by groovecoder** [tulsawebdevs/tulsawebdevs.org#28]
I'd like to get something up within a week or two so anyone clicking thru 200ok.us and/or techlahoma.org sites will see something. I might throw up a quick GitHub Pages site with links and widgets to our events calendars and our other online channels. Should I go ahead and do that while we wait for this code?

**Comment by nearbycoder** [comment]
Currently not sure about this. I talked with @destos yesterday about the project and seeing how far along the project was. I suggested moving it over to ruby on rails api with react for the frontend as I am more familiar with those technologies and could contribute more. This weekend I was working on the home page design and this is what I came up with http://imgur.com/a/yjp4Z.

**Comment by groovecoder** [comment]
FWIW, I threw this GitHub pages temporary site up so visitors can at least find our channels and events: http://tulsawebdevs.org/

**Comment by groovecoder** [comment]
I like the design. @chimchim237 has a new Tulsa Web Devs logo for us already ...

**Comment by nearbycoder** [comment]
Ya it was a placeholder waiting for something to replace it. If you have a link to the image/svg let me know.

**Comment by chimchim237** [comment]
logo assets:
https://www.dropbox.com/sh/jzu0pnf955qktze/AAB0reKawclmEz1ZWVPGZrxla?dl=0

**Comment by nearbycoder** [comment]
Thank you @chimchim237 the comps have been updated with the logo.
![tulsawebdev](http://i.imgur.com/JSJi5jY.png)
![tulsawebdev mobile](http://i.imgur.com/1b3CxZJ.png)

**Comment by jgmize** [comment]
> FWIW, I threw this GitHub pages temporary site up so visitors can at least find our channels and events: http://tulsawebdevs.org/

Thanks for getting something working now, and not waiting for dynamic backends to be "finished", @groovecoder. I think we should strongly consider continuing to use a static home page going forward that always works regardless of the availability of a dynamic backend. 

I think @destos has done a lot of work on this repo already, and if/when he feels it's ready I look forward to the results, but those features can be either pulled in via js if they need to be on the homepage, with graceful degradation if the backend is not responding, or they can simply be linked to from the homepage.

> Currently not sure about this. I talked with @destos yesterday about the project and seeing how far along the project was. I suggested moving it over to ruby on rails api with react for the frontend as I am more familiar with those technologies and could contribute more. This weekend I was working on the home page design and this is what I came up with http://imgur.com/a/yjp4Z.

@nearbycoder I'd really rather avoid a debate on the merits of rails vs django for a dynamic backend, but if we can agree on a static homepage I'm all for bringing your frontend work and @chimchim237's design work into production. Would you be willing to turn your work so far into a PR against https://github.com/groovecoder/tulsawebdevs for now, and we can discuss future backend and repo changes at the next meeting?

**Comment by nearbycoder** [comment]
No problem. New to the community so I didn't know there was a debate going on between frameworks/languages. Sure I can put together a PR against that repo. Should I comment out all the nav links until those pages are done and just have a static homepage? 

For now I am using a simple markdown javascript library to turn the meetings notes into html https://github.com/chjj/marked is the library. This will make it easy to just grab the markdown notes and toss them into a meeting div. 

As well I was considering adding some Admins to the frontpage so that any new members could contact for questions. Below is a comp of this also if you would like this included let me know the three Admins that you would wanted included in the PR or if you would like it removed. ![comp admins](http://i.imgur.com/QvSpePs.png)

**Comment by groovecoder** [comment]
Cool! I love the meeting-notes-in-markdown-to-html-pages-on-the-site idea. I recently moved meeting notes to wiki pages on GitHub:

https://github.com/tulsawebdevs/tasks/wiki

I'm also inclined to stay away from a backend besides plain Jekyll & GitHub pages until we have something that needs it.

**Comment by destos** [comment]
How am I not getting notified about this convo… Reviewing. :/

**Comment by destos** [comment]
I'm all for moving to a static site for simplicity. This project was attempting to solve a couple of problems we didn't have. I thought having interesting features with a backend would incite people to participate and add to it, especially since a lot of us in the group share the same backend knowledge. I was wrong. I'm assuming time issues. I'm grateful for those that have helped like @nearbycoder and @jgmize. In the end, I just couldn't drive it forward and allocated my free time to other things.

Related to efforts with the static site, let's have a discussion for a few minutes before or after our next meeting to re-align ( manager speak ) our previous IA plans and start doing some wireframes. ( IA == the Information Architecture )

@nearbycoder The designs you've come up with are a great start but aren't jiving with me. A lot of the design elements feel dated and is very monotone. I've fallen into the same design errors before.

I would like to try and get our designers to come up with some complimentary colors we can use for dividers and background elements/images and maybe even assign them to specific uses/concepts of the IA.

_A quick triad layout of the base color:_
<img width="767" alt="screen shot 2016-01-12 at 10 57 47 pm" src="https://cloud.githubusercontent.com/assets/136456/12285459/02138e40-b980-11e5-8b8d-efe3a6c615cb.png">

@chimchim237 is that something you and Dave already did? Also, I don't think we had a download of all the branding efforts that went into the logo creation. It would be nice to get those somewhere we can read to help drive the direction of the design.

Well, that's my input. Does it sound like a good direction to go?

**Comment by nearbycoder** [comment]
@destos sounds gouda just let me know when we have a layout mockup and I can help put that together.

**Comment by groovecoder** [comment]
I added website topic to the next Meta meeting agenda:

https://github.com/tulsawebdevs/tasks/wiki/January-2016-Meeting

**Comment by destos** [comment]
@nearbycoder I'm not kicking you off layout mockups, I would like to start with what we need in them then go wireframing from there. Which is why I would like to be aware of branding efforts as that can dictate verbiage, call to actions and colors.

Will you be coming to Monday's meeting?

**Comment by nearbycoder** [comment]
@destos yes I will be there. Would like to get more of a feel for what we are going for. I am really not use to user stories that was hard for me to decipher. As well reviewing the site through archive.org didn't give me much to go on. I also didn't find very much imagery that was at a resolution we could use for the site. Of course I may be looking in all the wrong places as well.

**Comment by destos** [comment]
A lot of that just comes from communicating more and getting resources like images from existing members or previous meetings and presenters.

All of what I've been saying adds time to getting something even basic up. So I'm unsure if I'm helping get a better result or just prolonging the whole effort. Maybe we take a piecemeal approach and just start hacking on it with your initial mock and improve from there. 

@groovecoder related to twd.org, If we're potentially re-branding TWD into something else should we even continue with this? Or is that far enough off that it won't matter for now?

**Comment by nearbycoder** [comment]
Ya that makes sense. I went to one of the developer meet ups last year but I planned to go to most if not all this year.

**Comment by groovecoder** [comment]
I'm sure we'll have use for our own TWD brand. It has a lot of credibility
in town. E.g., city councilors still refer to Tulsa Web Devs rather than
Code for Tulsa.
